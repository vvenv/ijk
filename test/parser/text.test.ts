import { expect, test } from 'vitest';
import { parse } from './__helper';

test('empty', () => {
  expect(parse(``)).toMatchSnapshot();
});

test('html tags', () => {
  expect(parse(`<foo>foo</foo>`)).toMatchSnapshot();
});

test('quotes', () => {
  expect(parse(`"'foo'"`)).toMatchSnapshot();
});

test('line break feed', () => {
  expect(parse(`\nfoo\n`)).toMatchSnapshot();
});
