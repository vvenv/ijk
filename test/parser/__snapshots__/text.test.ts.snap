// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`empty 1`] = `""use strict";let s = "";return s;"`;

exports[`html tags 1`] = `""use strict";let s = "";s+="<foo>foo</foo>";return s;"`;

exports[`line break feed 1`] = `""use strict";let s = "";s+=" foo ";return s;"`;

exports[`quotes 1`] = `""use strict";let s = "";s+="\\"'foo'\\"";return s;"`;
